### ISSUE TITLE
Dance of Doom no targetable AoA creature power

### ISSUE THREAD

**Issue by Tuvak4X** [keyteki/keyteki#318]
I played Dance of Doom and the game didnt let me choose creature power 3 and there was only one creature with power 3 in play and it was from the new set AoA.

**Comment by Achocholacek** [comment]
Do you remember what your options were? Was it everything *but* 3?

**Comment by Tuvak4X** [comment]
If I remember correctly all power numbers of creatures in play were displayes to choose from, but the 3 was missing.

**Comment by romanogit** [comment]
A side note on this one. Dance of Doom only kills up to 13 and, in theory, it should allow any number ot be selected and not restrict in the GUI.

```
for(let i = 1; i < 13; i++) {
            choices[i.toString()] = ability.actions.destroy(context => ({
                target: context.game.creaturesInPlay.filter(card => card.power === i)
            }));
        }
```

There are several effects that can change the creatures power, but, again, it should not restrict power selection.

@cryogen : I wonder if we can use new dropdown instead of showing 16 buttons.

**Comment by romanogit** [comment]
Fixed by #970
